docs/en/observability/apm/sampling.asciidoc

```diff
@@ -45,6 +45,33 @@ Again, the sample rates of `Service B` and `Service C` are ignored.
 
 image::./images/dt-sampling-example-2.png[Distributed tracing and head based sampling example two]
 
+In addition to setting the sample rate, you can also specify what trace continuation strategy to use.
+This can be especially helpful if:
+
+* An Elastic-monitored service is receiving requests with `traceparent` headers from unmonitored or third-party monitored services.
+* An Elastic-monitored service is publicly exposed, and does not want tracing data to possibly be spoofed by user requests.
+
+The default continuation strategy is `continue`, which will behave similar to the examples above.
+
+The example below uses the `restart` continuation strategy in which a new trace is started with every service.
+// Is this true? Is the opposite true?
+`Service A` initiates four transactions, and because it is a third-party monitored service none of the transactions will be sampled.
+`Service B` and `Service C` are Elastic-monitored services with sample rates of `.5` (`50%`) and `1` (`100%`) respectively.
+Because these are treated as separate traces when using the `restart` continuation strategy,
+the sample rate for each service is applied, unlike the examples above.
+
+image::./images/dt-sampling-continuation-strategy-restart.png[Distributed tracing and head based sampling with restart continuation strategy]
+
+The next example uses the `restart_external` continuation strategy in which a new
+trace is only started if the previous service was not Elastic-monitored.
+`Service A` is an Elastic-monitored service initiates four transactions with a sample rate of `.25` (`25%`).
+// Is `Service B` part of the trace that starts in `Service A`?
+Because `Service B` is a third-party monitored service that will be the end of the traces started in `Service A`.
+`Service C` will initiate four transactions that which start new traces with a sample rate of `.5` (`50%`).
+Because `Service D` is also Elastic-monitored service, the sample rates of `Service D` is ignored.
+
+image::./images/dt-sampling-continuation-strategy-restart_external.png[Distributed tracing and head based sampling with restart_external continuation strategy]
```

Comment:
In the case of `sample 1`

Because `Service A` is from Elastic and Service B is monitored the `es=` tracestate MUST be propagated by the vendor in `Service B` see: https://www.w3.org/TR/trace-context/#mutating-the-traceparent-field

Service B has to adhere to our sampling decision so I'd expect `Service C` to continue `sample 1`

Reply:
Maybe rename `sample` to `trace` ?

Reply:
Is there a better example that would illustrate how `restart_external` works? Like if `Service B` is just unmonitored, would the trace stop at `Service B` and restart at `Service C`?

Reply:
Yes it would since `B` would not forward tracestate.

I think its important to drive home here that `restart_external` is great for cases where the there is a gap in monitored services that are not all Elastic. 

Typically you only want to set `restart_external` on the first incoming hop that is elastic.

Reply:
@Mpdreamz  does this look accurate? 

![image](https://github.com/user-attachments/assets/faa22535-4c68-48b1-892a-a4503a1d5859)

Reply:
This looks great @colleenmcginnis!